Mathlib/Geometry/Manifold/MFDeriv/Defs.lean

```diff
@@ -127,6 +127,7 @@ differentiable functions between manifolds. -/
 def DifferentiableWithinAtProp (f : H → H') (s : Set H) (x : H) : Prop :=
   DifferentiableWithinAt 𝕜 (I' ∘ f ∘ I.symm) (I.symm ⁻¹' s ∩ Set.range I) (I x)
 
+variable {I I'} in
```

Comment:
The current variable management in this file technically *works* - but I would find it clearer to flip the default: make `I` and `I'` implicit in the variable block, and explicit in each definition. What do you think?

Reply:
Most things in this file are definitions, so `I` and `I'` would need to be made explicit for most statements. But still, I agree with you that it's clearer to follow an overall design, so I've made the change.